rococo-parachains/runtime/src/lib.rs

```diff
@@ -376,6 +379,34 @@ impl cumulus_ping::Config for Runtime {
 	type XcmSender = XcmRouter;
 }
 
+parameter_types! {
+	pub const AssetDeposit: Balance = 100;
+	pub const ApprovalDeposit: Balance = 1;
+	pub const StringLimit: u32 = 50;
+	pub const MetadataDepositBase: Balance = 10;
+	pub const MetadataDepositPerByte: Balance = 1;
+	pub const UnitBody: BodyId = BodyId::Unit;
+}
```

Comment:
Even for Rococo, probably not the right units? Should at least use some kind of `DOLLARS` and stuff.